packages-internal/test-utils/src/index.ts

```diff
@@ -15,9 +15,12 @@ export {} from './initMatchers';
 export * as fireDiscreteEvent from './fireDiscreteEvent';
 export * as userEvent from './userEvent';
 export { default as flushMicrotasks } from './flushMicrotasks';
+export { default as reactMajor } from './reactMajor';
 
 /**
  * Set to true if console logs during [lifecycles that are invoked twice in `React.StrictMode`](https://reactjs.org/docs/strict-mode.html#detecting-unexpected-side-effects) are suppressed.
  * Useful for asserting on `console.warn` or `console.error` via `toErrorDev()`.
+ * TODO: Refactor to use reactMajor when fixing the React 17 cron test.
```

Comment:
Why do we need to wait to use `reactMajor`?

Reply:
I don't want to include React 17 related changes in this PR because it would mean having to switch between three different React versions, but I created an issue for it: https://github.com/mui/material-ui/issues/43153

Reply:
Nice 👍